src/fides/api/service/messaging/message_dispatch_service.py

```diff
@@ -494,6 +496,40 @@ def _build_email(  # pylint: disable=too-many-return-statements, too-many-branch
             template_variables=variables,
         )
 
+    if action_type == MessagingActionType.MANUAL_TASK_DIGEST:
+        variables = {
+            "vendor_contact_name": body_params.vendor_contact_name,
+            "organization_name": body_params.organization_name,
+            "portal_url": body_params.portal_url,
+            "imminent_task_count": body_params.imminent_task_count,
+            "upcoming_task_count": body_params.upcoming_task_count,
+            "company_logo_url": body_params.company_logo_url,
```

Comment:
We don't have to do it for this PR, but I think it'd be good to tag @jack-gale-ethyca and see if there is a way to make these variables more discoverable in the UI. Maybe an auto-suggest after you type the double-underscores?